services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -817,6 +816,56 @@ class FlowCrudSpec extends HealthCheckSpecification {
         flows.each { flowHelper.deleteFlow(it) }
     }
 
+    def "System takes isl time_unstable info into account while creating a flow"() {
+        given: "Two active not neighboring switches with two possible paths at least"
+        def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find {
+            it.paths.unique { it.first().portNo }.size() > 1
+        } ?: assumeTrue("No suiting switches found", false)
+
+        and: "Detect two available paths"
```

Comment:
I don't understand what I should do in this step

Reply:
the step was renamed